packages/loot-core/src/types/api-handlers.ts

```diff
@@ -186,4 +194,28 @@ export interface ApiHandlers {
   'api/rule-update': (arg: { rule: RuleEntity }) => Promise<RuleEntity>;
 
   'api/rule-delete': (id: string) => Promise<boolean>;
+
+  'api/schedule-create': (arg: {
+    schedule: Partial<ScheduleEntity>;
+    conditions: Partial<RuleConditionEntity>[];
+  }) => Promise<string>;
+
+  'api/schedule-update': (arg: {
+    id: string;
+    conditions: Partial<RuleConditionEntity>[];
+    resetNextDate?: boolean;
+  }) => Promise<string>;
```

Comment:
Same comment on the other functions:
```suggestion
  'api/schedule-update': (arg: {
    id: ScheduleEntity['id'];
    conditions: Partial<RuleConditionEntity>[];
    resetNextDate?: boolean;
  }) => Promise<ScheduleEntity['id']>;
```